simulator/src/test/java/com/hedera/block/simulator/BlockStreamSimulatorTest.java

```diff
@@ -17,17 +17,22 @@
 package com.hedera.block.simulator;
 
 import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Wildcard imports are not good.  We should avoid them (and set IDE settings to not use them).

Reply:
Yes, I thought I have my IDE set to do that 🤔 
good catch, easy to overlook it since is a Test class of the simulator (where we are a bit more relaxed)

Reply:
I will fix it in the next PR, already working on that 😅  since I need to add Simulation Modes that modifies exit conditions, modes to the rate of streaming, etc... so we can do more kinds of tests.